### DISCUSSION TITLE
What's new in every release?

### DISCUSSION THREAD

**Discussion by stx-hseow** [Seagate/cortx#1541] [Q&A]
The main difference I could tell from every other release is that the OVA file size is getting larger and larger.

Besides that, is there any plan to include a change log (or feature update) to show what's new & exciting (or fixed) during every release?

**EDIT**: My bad, I realized that they are written in detail at [CHANGELOD.md](https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-7/CHANGELOG.md)

**Comment by hessio** [comment]
Yes every release comes with a [CHANGELOG.md](https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-7/CHANGELOG.md) file detailing all the changes.
